docs/content/querying/sql.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: doc_page
+---
+# SQL Support for Druid
```

Comment:
I want to add more to this page in the future. I think it is important it has its own section